vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -33,30 +54,17 @@
         git:
           repo: 'https://github.com/metal3-io/baremetal-operator.git'
           dest: "/home/{{ IMAGE_USERNAME }}/baremetal-operator"
-
-        # Replacing the name of the bridge
-      - name: Replace the Provisioning Interface in ironic_ci.env
-        lineinfile:
-          path: /home/{{ IMAGE_USERNAME }}/baremetal-operator/deploy/ironic_ci.env
-          regexp: "^PROVISIONING_INTERFACE="
-          line: "PROVISIONING_INTERFACE={{ IRONIC_ENDPOINT_BRIDGE }}"
-
-        # Install ironic
-      - name: Install ironic
-        shell: "bash /home/{{ IMAGE_USERNAME }}/baremetal-operator/tools/run_local_ironic.sh"
-        environment:
-          CONTAINER_RUNTIME: "{{ CONTAINER_RUNTIME }}"
-        become: yes
-        become_user: root
+        when: CAPI_VERSION == "v1alpha3"
```

Comment:
You could use this condition once for the entire block so that you don't have to duplicate it in the next task under the same block.

Reply:
Done. Thanks